src/main.c

```diff
@@ -606,12 +606,16 @@ static int listener_setup_ssl(h2o_configurator_command_t *cmd, h2o_configurator_
                                                                 : "share/h2o/fetch-ocsp-response";
         if (ocsp_update_interval != 0) {
             switch (conf.run_mode) {
-            case RUN_MODE_WORKER:
+            case RUN_MODE_WORKER: {
+                int err;
                 ssl_config->ocsp_stapling.interval =
                     ocsp_update_interval; /* is also used as a flag for indicating if the updater thread was spawned */
                 ssl_config->ocsp_stapling.max_failures = ocsp_max_failures;
-                pthread_create(&ssl_config->ocsp_stapling.updater_tid, NULL, ocsp_updater_thread, ssl_config);
-                break;
+                err = pthread_create(&ssl_config->ocsp_stapling.updater_tid, NULL, ocsp_updater_thread, ssl_config);
+                if (err != 0) {
+                    fprintf(stderr, "[OCSP Stapling] pthread_create for ocsp_updater_thread failed. errno:%d\n", err);
```

Comment:
Even if `pthread_create()` for `ocsp_updater_thread()` fails, `h2o` can continue to serve. But I'm a little worried whether  `h2o` should continue to serve in this case. How about you?